low-level/files/opt/seagate/sspl/setup/platform_based_config_update.py

```diff
@@ -53,3 +53,24 @@ def update_sensor_info(config_index, node_type, enclosure_type):
         Conf.set(config_index, '%s>monitor' % sect, value)
 
     Conf.save(config_index)
+
+def update_monitored_services(config_index, node_type):
+    """Based on node type exclude the services from list
+    of monitored services.
+    """
+    # Align to node type as used in sspl.conf SERVICEMONITOR section
+    vm_types = ["virtual", "vm"]
+    node_type = "vm" if node_type.lower() in vm_types else "hw"
+
+    monitored_services = Conf.get(
+        config_index, "SERVICEMONITOR>monitored_services", [])
```

Comment:
log enough from here with info log level
original monitored services list
excluded monitored services list for env 'vm/hw'
effective monitored services list

Reply:
With new commit, I iIntroduced the logging in Platform.get_effective_monitored_services()

Reply:
Done.